src/main/java/software/amazon/qldb/QldbDriverBuilder.java

```diff
@@ -61,7 +63,9 @@
     /**
      * <p>Specify the low level QLDB session builder that should be used for accessing QLDB.</p>
      *
-     * <p>Note that the user agent suffix and retry count will be set on the specified session builder.</p>
+     * <p>Note that the user agent suffix and retry count will be set on the specified session builder. And the
+     * http client will be overwritten. To pass customized http client to the the {@link QldbDriver},
+     * use {@link #httpClientBuilder(SdkHttpClient.Builder)}</p>
```

Comment:
Minor grammatical issues:


Note that the user agent suffix and retry count will be set on the specified session builder, and the http client will be overwritten. To pass a customized http client to the the {@link QldbDriver}, use {@link #httpClientBuilder(SdkHttpClient.Builder)}.